### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by AnuragSahu11**
https://deploy-preview-2--wizardly-torvalds-416226.netlify.app/pages/cart.html
*Added cart management page

### CODE REVIEW

File: pages/cart.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+@media only screen and (max-width: 768px) {
+  .cart-summary {
+    width: 80%;
+  }
+}
+@media only screen and (max-width: 1124px) {
+  .cart-card {
+    width: 100%;
+  }
+}
```
Comment (Hamdrive) [inline pages/cart.css:13]: Preferable to stick to most commonly used screen sizes for media queries

**Review by Hamdrive — COMMENTED** [review]
Good work done, suggestion for responsiveness given.

**Review by abhij1607 — APPROVED** [review]
Very Clean code and UI, great work Anurag

File: pages/cart.css (hunk 1 above)
Comment (AnuragSahu11) [reply pages/cart.css:13]: Yup

**Pull request merged**